contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Team Core Analytics
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- [Eric Duong (Team Lead, Full Stack Engineer)](/handbook/people/team/#eric-duong-software-engineer)
+- [Buddy Williams, Full Stack Engineer](/handbook/people/team/#buddy-williams-software-engineer)
+
+## Mission
+
+Make PostHog the best and most performant product analytics platform.
```

Comment:
Best seems too ambiguous. I think we want the goal around performance, data quality and functionality?